python/tidy/servo_tidy/tidy.py

```diff
@@ -642,6 +642,23 @@ def check_wpt_lint_errors(files):
             yield ("WPT Lint Tool", "", "lint error(s) in Web Platform Tests: exit status {0}".format(returncode))
 
 
+def check_dep_license_errors(progress):
+    for root, directories, filenames in os.walk(".cargo"):
```

Comment:
It'd be better if we print something like `'Checking licenses...'` before beginning the iterator.